Source/RiveRenderer/Private/Platform/pls_render_context_rhi_impl.hpp

```diff
@@ -0,0 +1,245 @@
+#pragma once
+#include <chrono>
+
+#include "Shaders/ShaderPipelineManager.h"
+
+THIRD_PARTY_INCLUDES_START
+#undef PI
+#include "rive/renderer/rive_renderer.hpp"
+#include "rive/renderer/rive_render_image.hpp"
+#include "rive/renderer/buffer_ring.hpp"
+THIRD_PARTY_INCLUDES_END
+
+namespace rive
+{
+namespace gpu
+{
+class RIVERENDERER_API RenderTargetRHI : public RenderTarget
```

Comment:
Unreal is unfriendly to classes nested inside of namespaces (you won't see Unreal-proper classes inside of namespaces). I suggest not embedding Unreal classes inside of namespaces.